traffic_ops/traffic_ops_golang/cdn/cdns.go

```diff
@@ -162,36 +176,75 @@ func (cdn *TOCDN) Delete() (error, error, int) {
 	return api.GenericDelete(cdn)
 }
 
-func selectQuery() string {
+func selectQuery(apiVersion *api.Version) string {
+	var ttlOverrideColumn string
```

Comment:
Since this code block is being replicated in a couple of different places, would you mind making it into it's own function?

Reply:
Made it into its own function in d6c88722e4